File: Cocoa Programming/Cocoa Pr L67 PresentingVC/Cocoa Pr L67 PresentingVC/ModalViewController.m
//
//  ModalViewController.m
//  Cocoa Pr L67 PresentingVC
//
//  Created by Michele Galvagno on 08/07/22.
//

#import "ModalViewController.h"

@interface ModalViewController ()

@end

@implementation ModalViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [[self.view layer] setBackgroundColor:NSColor.systemYellowColor.CGColor];
    [[self.view layer] setCornerRadius:5];
}

- (IBAction)dismissVC:(NSButton *)sender {
    [self dismissController:self];
}

@end

File: Cocoa Programming/Cocoa Pr L67 Swift/Cocoa Pr L67 Swift/ModalViewController.swift
//
//  ModalViewController.swift
//  Cocoa Pr L67 Swift
//
//  Created by Michele Galvagno on 07/07/22.
//

import Cocoa

class ModalViewController: NSViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        view.layer?.backgroundColor = NSColor.systemYellow.cgColor
        view.layer?.cornerRadius = 5
    }
    
    @IBAction func dismissVC(_ sender: NSButton) {
        dismiss(self)
    }
}
